framework/BaseYii.php

```diff
@@ -138,11 +138,13 @@ public static function getAlias($alias, $throwException = true)
 
         if (isset(static::$aliases[$root])) {
             if (is_string(static::$aliases[$root])) {
-                return $pos === false ? static::$aliases[$root] : static::$aliases[$root] . substr($alias, $pos);
+                $path = $pos === false ? static::$aliases[$root] : static::$aliases[$root] . substr($alias, $pos);
+                return strncmp($path, '@', 1) ? $path : static::getAlias($path);
```

Comment:
It's already used without `===0` on other places. That's why I've used the same pattern. If we change this we should change it everywhere.